stew/varints.nim

```diff
@@ -113,11 +113,12 @@ func feedByte*(p: var VarintParser, b: byte): VarintState =
   ##     has been exceed. The supplied input can be considered invalid.
   ##
   const maxShift = maxBits type(p)
+  type IntType = p.IntType
```

Comment:
this is some new bug workaround?

Reply:
Yup

Reply:
dcdbaaeaf3d3d10ac6f0dda4911f0f42e0675a5a
https://github.com/nim-lang/Nim/issues/20645